docs/source/user/index.rst

```diff
@@ -18,6 +18,7 @@ User Manual
    withobjmode.rst
    performance-tips.rst
    threading-layer.rst
+   cli.rst
```

Comment:
I've put the "Command line interface" page at the end of the user manual, before the "Troubleshooting and Tips":

![Screenshot 2019-05-07 at 16 20 34](https://user-images.githubusercontent.com/852409/57307005-15aca080-70e4-11e9-91af-2faaa6e3f3d6.png)

Another option would be to put it after "Installation", but that would put it at the front, and really the CLI isn't very useful, so IMO the less prominent place at the end of the user manual is better.

Thoughts?

Reply:
I agree, this is a good location.